test/container/kafka_container_test.exs

```diff
@@ -239,6 +255,35 @@ defmodule Testcontainers.Container.KafkaContainerTest do
 
       assert response.value == "hey"
     end
+
+    test "with multiple connected nodes" do
+      {:ok, zookeeper} = start_external_zookeeper()
+      {:ok, kafka1} = start_kafka_with_external_zookeeper(zookeeper, 1, 9092)
+      {:ok, kafka2} = start_kafka_with_external_zookeeper(zookeeper, 2, 9093)
+
+      uris_one = [{"localhost", Container.mapped_port(kafka1, 9092) || 9092}]
```

Comment:
I dont remember who added this, but we can't continue with this shenanigan. I mean its saying "we dont know if the port will be mapped so lets just say that its 9092 just in case as a backup". A much better approach would be to accept the bad behaviour and just say `uris_one = [{"localhost", 9092]` or remove the `|| 9092`. Could be me who is the originator of this anti pattern, and if so I bow down in the dust of shame. But can you can clean it up while you are in the code ? I think it happens more than one place 🙈  and I mean it's a test for sure, but It makes the test harder to read

Reply:
Sure thing, added & rebased.